packages/module/release.config.js

```diff
@@ -1,8 +1,5 @@
 module.exports = {
-  branches: [
-    'do-not-delete',
-    { name: 'main', channel: 'prerelease', prerelease: 'prerelease' },
-  ],
+  branches: ['do-not-delete', { name: 'main', channel: 'prerelease', prerelease: 'prerelease' }],
```

Comment:
`prettier` build from a [previous build](https://github.com/patternfly/react-topology/pull/266/checks?check_run_id=35214108931) asked me to make this change in this stage of CI:

```
yarn run v1.22.22
$ eslint . --ext .js,.jsx,.ts,.tsx --color

/home/runner/work/react-topology/react-topology/packages/module/release.config.js
  2:14  error  Replace `⏎····'do-not-delete',⏎····{·name:·'main',·channel:·'prerelease',·prerelease:·'prerelease'·},⏎··` with `'do-not-delete',·{·name:·'main',·channel:·'prerelease',·prerelease:·'prerelease'·}`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
  ```